package.json

```diff
@@ -33,6 +33,9 @@
   },
   "dependencies": {
     "adiff": "^0.2.13",
+    "react": "^15.3.1",
```

Comment:
I don't think that's a good idea. It adds a significant weight to the serialized diff page.